tests/flytekit/unit/models/test_concurrency.py

```diff
@@ -0,0 +1,41 @@
+from flyteidl.admin import launch_plan_pb2 as _launch_plan_idl
+
+from flytekit.models.concurrency import ConcurrencyLimitBehavior, ConcurrencyPolicy
+
+
+def test_concurrency_limit_behavior():
+    assert ConcurrencyLimitBehavior.SKIP == _launch_plan_idl.ConcurrencyLimitBehavior.SKIP
```

Comment:
```suggestion
    assert ConcurrencyLimitBehavior.SKIP == _launch_plan_idl.CONCURRENCY_LIMIT_BEHAVIOR_SKIP
```

All `_launch_plan_idl.ConcurrencyLimitBehavior.SKIP` below should be changed to `_launch_plan_idl.CONCURRENCY_LIMIT_BEHAVIOR_SKIP`, then the test should pass